force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -259,11 +307,11 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Integer performDivisionAndCast(Double a, Double b) {
         // Perform the division and cast (round down) off the result.
-        Double divisionResult = null;
+        Double divisionResult = Math.floor(a / b);
 
         // Write the code for type casting the divisionResult to an Integer
-        Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        Integer roundedResult = divisionResult.intValue();
```

Comment:
Nicely done! Another method to convert to an Integer is to cast like this:
I`nteger roundedResult = (Integer) divistionResult;`